client/src/components/Workflow/WorkflowNavigationTitle.vue

```diff
@@ -102,7 +103,8 @@ const workflowImportTitle = computed(() => {
                 <div class="d-flex portlet-header align-items-center">
                     <div class="flex-grow-1" data-description="workflow heading">
                         <div class="px-1">
-                            <FontAwesomeIcon :icon="faSitemap" />
+                            <FontAwesomeIcon v-if="props.invocationRunning" :icon="faSpinner" spin />
```

Comment:
Since we mainly use the spinner icon as a loading/fetching indicator, I'd recommend using the spinning gear icon instead here

Reply:
Ok, we can use the gear icon. I wonder if we have some example somewhere for an icon for something currently running?

Also, I did think about making the workflow (faSitemap) icon spin as well 😅🤷‍♂️?

Reply:
| Current `faSpinner` |
| ---- |
| ![firefox_hq8OtFiFTZ](https://github.com/user-attachments/assets/db722f83-5b15-48f0-9fb9-a00276a029f7) |

| `faCog` |
| ---- |
| ![firefox_en3AmtlMKY](https://github.com/user-attachments/assets/451878ed-f82e-48fb-8cf5-d44b17e6f737) |

| `faSitemap` |
| ---- |
| ![firefox_YBz3W6uJgZ](https://github.com/user-attachments/assets/62940750-12e7-48ef-b915-a35a6fc73dff) |

Reply:
We use the same spinner for running jobs, and it has a reasonable historical meaning here—I don't mind it in this context.

I am against the cog here. While it looks cool, we already use it for a completely different purpose (settings, options, etc.), as is the common usage these days and historically in Galaxy.

faSitemap is hilarious—maybe for April Fools.

All that said, I'm not sure anything needs to be spinning here—the rest of this interface makes it pretty clear when the workflow is still running with the bars, I think.

Reply:
Done. Removed the spinner entirely..